client/src/components/TaskDurationBarChart/TaskDurationBarChart.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styles from "./TaskDurationBarChart.module.css";
+import PropTypes from "prop-types";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  ReferenceLine,
+  ResponsiveContainer,
+} from "recharts";
+
+export const TaskDurationBarChart = ({ taskResponseData }) => {
+  const taskDurationDifferences = [];
```

Comment:
This part including the for loop can be placed inside a function (this will help better encapsulate our logic and if we were to add more code, we won't be as confused since it's grouped up in a function)

Reply:
Once grouped in a function, you'll just have to return `taskDuractionDifference`

Reply:
Moved logic into the function `computeDurationDifference`.